tools/license_header.py

```diff
@@ -110,6 +110,11 @@
                # This file
                'tools/license_header.py',
 
+               # Dual-Licensed under Apache 2.0 and Nvidia BSD-3
+               'python/mxnet/onnx/mx2onnx/_export_onnx.py',
+               'python/mxnet/onnx/mx2onnx/_op_translations/_op_translations_opset12.py',
+               'python/mxnet/onnx/mx2onnx/_op_translations/_op_translations_opset13.py',
```

Comment:
Dual licensed files should also be mentioned in the root LICENSE file. However, as these files start with AL2 License text, why do you need to include them in `license_header.py`? If the license test fails, would it succeed if Line 17 (the line after end of AL2 text would be empty instead of `#`?

Reply:
the `op_translation` files were originally developed by Nvidia I believe so they were dual licensed https://github.com/apache/incubator-mxnet/blob/97a2b609042e8594c6b1ad655a74f7eef585cfd0/python/mxnet/onnx/mx2onnx/_op_translations/_op_translations_opset12.py. Let me also mention them in root LICENSE